opm/simulators/flow/GenericCpGridVanguard.hpp

```diff
@@ -22,15 +22,15 @@
 */
 /*!
  * \file
- * \copydoc Opm::EclCpGridVanguard
+ * \copydoc Opm::CpGridVanguard
```

Comment:
Should this reflect the class name?

Reply:
It's used for copying the doxy from the Opm::CpGridVanguard class. The generic class always used the same doxy as the base class, ie. nothing changes.